src/web/make-graph.rkt

```diff
@@ -56,13 +56,21 @@
   (define repr-bits (representation-total-bits repr))
   (define ctx (test-context test))
   (define identifier (test-identifier test))
-  (match-define (improve-result preprocessing pctxs start target end bogosity) backend)
+  (match-define (improve-result preprocessing pctxs start targets end bogosity) backend)
 
   (match-define (alt-analysis start-alt _ start-error) start)
   (define start-cost (alt-cost start-alt repr))
-  (define target-alt (and target (alt-analysis-alt target)))
-  (define target-error (and target (alt-analysis-test-errors target)))
-  (define target-cost (and target (alt-cost target-alt repr)))
+
+  (define target-alt-list (filter identity targets))
```

Comment:
This wouldn't be necessary if we filtered out the invalid entries using `#:when` like suggested above.